src/application/shell/command/Cd.cpp

```diff
@@ -30,6 +30,8 @@ void Cd::execute(Util::Array<String> &args) {
     }
 
     if(parser.getUnnamedArguments().length() < 1) {
+        /* For cd command with no arguments will change the current working directory to the root directory */
+        shell.setCurrentWorkingDirectory(Directory::open("/"));
```

Comment:
As far as we know, executing cd without arguments on a Unix-based system, does not do anything. That is why, we implemented the command this way. We think, this is a more natural behaviour than changing to the root directory.